examples/typical_app/.happy/terraform/envs/rdev/main.tf

```diff
@@ -76,4 +84,9 @@ module "stack" {
   // tasks can be utilized to run post-deployment tasks such as database migrations or deletions
   tasks = {
   }
+
+
+  providers = {
```

Comment:
this is required for TLS certificates. They have to exist in this region for some reason